executor/recover_test.go

```diff
@@ -295,6 +297,14 @@ func TestRecoverTableMeetError(t *testing.T) {
 func TestRecoverClusterMeetError(t *testing.T) {
 	store := testkit.CreateMockStore(t)
 	tk := testkit.NewTestKit(t, store)
+	ts, _ := tk.Session().GetStore().GetOracle().GetTimestamp(context.Background(), &oracle.Option{})
+	flashbackTs := oracle.GetTimeFromTS(ts)
+
+	injectSafeTS := oracle.GoTimeToTS(flashbackTs.Add(10 * time.Second))
+	require.NoError(t, failpoint.Enable("tikvclient/injectSafeTS",
+		fmt.Sprintf("return(%v)", injectSafeTS)))
+	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/expression/injectSafeTS",
```

Comment:
This is unnecessary since it's used to test the `tidb_bounded_staleness` function.

Reply:
It's used to inject the `SafeTS` when the underlying KV store is mocked. Ignore this comment, please.